vagrant/README.md

```diff
@@ -124,3 +124,11 @@ Now start things up, but specify the aws provider:
 Your instances should get tagged with a name including your hostname to make
 them identifiable and make it easier to track instances in the AWS management
 console.
+
+### Libvirt with kvm ###
+
+Install the `vagrant-libvirt` plugin to provide libvirt and kvm support:
+```vagrant plugin install vagrant-libvirt``` or ```yum install vagrant-libvirt``` for fedora >=21
+
+Start things up with specified provider:
+```vagrant up --provider=libvirt --no-parallel --no-provision && vagrant provision```
```

Comment:
Is --no-parallel required here? I know the AWS one needs it because it's not properly synchronized. Is the same true of the libvirt plugin?